app/src/main/java/com/rob729/quiethours/util/DayPicker.kt

```diff
@@ -0,0 +1,22 @@
+package com.rob729.quiethours.util
+
+import ca.antonious.materialdaypicker.MaterialDayPicker
+
+object DayPicker {
```

Comment:
Rename the file to Utils and also change the function name to some appropriate accordingly.
It is not logical to make a file for just a single function. Make a common file wherein future functions that are used in multiple files can be placed.

Reply:
Renamed